test/js/search/more-facets.test.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import * as ReactAll from 'react';
+import { mount } from 'enzyme';
+import * as Reach from '@reach/router'
+
+const fetch = require('node-fetch');
+
+import MoreFacetsButton from 'components/MoreFacetsButton';
+import StudySearchProvider from 'components/search/StudySearchProvider';
+import * as ScpAPI from 'lib/scp-api'
+
+const testFacets = [{
```

Comment:
Capturing our discussion on test strategy w.r.t. this approach of having mock data directly within the test and using [`const routerNav = jest.spyOn(Reach, 'navigate')`](https://github.com/broadinstitute/single_cell_portal_core/pull/434/files#diff-127e766ff6b693241ea844277b907d1dR43):

Upside:
* Tests are more self-contained in terms of readability

Downside:
* Will have false negatives upon breaking changes in other parts of our JS code, e.g. `scp-api.js`

In other words, this is towards to the unit test end of the spectrum.  

Tests in e.g. [`download.test.js`](https://github.com/broadinstitute/single_cell_portal_core/blob/b72b6048ef95adb7ae3622043b2070e48ab194c0/test/js/download.test.js) are closer to integration tests in that they cover other parts of the JS code (e.g. code in `scp-api.js`).  They're more complex to maintain, but slightly less susceptible to false negatives.

A diversity of approaches is worthwhile!